src/constants/YossiRestaurantsData.ts

```diff
@@ -0,0 +1,25 @@
+import { CardProps } from "../models/types";
+
+import onzaRestaurant from "../assets/images/onzaRestaurant.png";
+import kitchenMarketRestaurant from "../assets/images/kitchenMarketRestaurant.png";
+import mashyaRestaurant from "../assets/images/mashyaRestaurant.png";
+
+const YossiRestaurantsData: { title: string; cards: CardProps[] } = {
```

Comment:
this should contain the info of chef of the week so u should create a diff interface for it